tests/test_expertise_apiv2.py

```diff
@@ -211,8 +211,71 @@ def test_journal_request_v2(self, openreview_client, openreview_context, celery_
         assert req['entityA']['memberOf'] == 'TMLR/Action_Editors'
         assert req['entityB']['type'] == 'Note'
         assert req['entityB']['id'] == target_id
+
         openreview_context['job_id'] = job_id
 
+        # Make a request for TMLR/Reviewers
+        response = test_client.post(
+            '/expertise',
+            data = json.dumps({
+                    "name": "test_run",
+                    "entityA": {
+                        'type': "Group",
+                        'memberOf': "TMLR/Reviewers",
+                    },
+                    "entityB": { 
+                        'type': "Note",
+                        'id': target_id
+                    },
+                    "model": {
+                            "name": "specter+mfr",
+                            'useTitle': False, 
+                            'useAbstract': True, 
+                            'skipSpecter': False,
+                            'scoreComputation': 'avg'
+                    }
+                }
+            ),
+            content_type='application/json',
+            headers=openreview_client.headers
+        )
+        assert response.status_code == 200, f'{response.json}'
+        job_id = response.json['jobId']
+        time.sleep(2)
+        response = test_client.get('/expertise/status', query_string={'jobId': f'{job_id}'}).json
+        assert response['name'] == 'test_run'
+        assert response['status'] != 'Error'
+
+        # Query until job is complete
+        response = test_client.get('/expertise/status', query_string={'jobId': f'{job_id}'}).json
+        start_time = time.time()
+        try_time = time.time() - start_time
+        while response['status'] != 'Completed' and try_time <= MAX_TIMEOUT:
+            time.sleep(5)
+            response = test_client.get('/expertise/status', query_string={'jobId': f'{job_id}'}).json
+            if response['status'] == 'Error':
+                assert False, response[0]['description']
+            try_time = time.time() - start_time
+
+        assert try_time <= MAX_TIMEOUT, 'Job has not completed in time'
+        assert response['status'] == 'Completed'
+        assert response['name'] == 'test_run'
+        assert response['description'] == 'Job is complete and the computed scores are ready'
+
+        # Test for paper id query
+        response = test_client.get('/expertise/status', query_string={'paperId': target_id}).json['results']
+        assert len(response) == 2
+
+        # Test for paper id and member of query
+        response = test_client.get('/expertise/status', query_string={'paperId': target_id, 'memberOf': 'TMLR/Reviewers'}).json['results']
```

Comment:
I see that your are not passing the prefix "entityA" or "entityB", how do you know that memberOf is to filter entityA and not B?

Reply:
Ahh I thought that checking either entity might be enough but yeah it would probably be better if we can specify which entity to check, I'm making the change now